public/data/briankiariemwaniki.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Brian Kiarie Mwaniki",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!Passionate about software & coding!",
+  "avatar": "https://github.com/BrianKiarieMwaniki.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/BrianKiarieMwaniki",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Brian_mwaniki_k",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/brian-kiarie-097b1b20b/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started OpenSource",
+      "date": "August 2022",
+      "icon": "open-source",
+      "color": "#3bc2cb",
+      "description": "Started freelancing",
+      "url": "https://github.com/EddieHubCommunity"
```

Comment:
Hmm, not sure if this link should be here unless you're saying you started freelancing on EddieHub. 🤔


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Huh! Makes sense I will edit the file

Reply:
Thanks for pointing that out